src/components/cart.lineitem.less

```
@@ -51,13 +52,24 @@
     font-weight: bold;
     padding-bottom: 3px;
     width: 60%;
+    vertical-align: top;
   }
 
   .options-col {
     display: inline-block;
     width: 50%;
   }
 
+  .promotions-col {
+    display: inline-block;
+    width: 33%;
+    text-align: right;
```

To achieve the styles in the screenshot above:
```suggestion
    display: block;
    width: 33%;
    text-align: left;
```